x-pack/docs/en/security/security-manual-configuration.asciidoc

```diff
@@ -5,51 +5,28 @@
 ++++
 
 Security needs vary depending on whether you're developing locally on your
-laptop or securing all communications in a production environment. Because
-security needs vary, the following scenarios provide options for configuring
-the Elastic Stack.
+laptop or securing all communications in a production environment. Regardless
+of where you're deploying the {stack} ("ELK"), running a secure cluster is
+incredibly important to protect your data. That's why security is
+<<configuring-stack-security,enabled and configured by default>> in {es} 8.0
+and later.
 
-Each subsequent scenario builds on the previous one so that you can add
-additional security by building on the existing layer.
+If you want to use your own Certificate Authority (CA) or would rather manually
+configure security, the following scenarios provide steps for configuring TLS
+on the transport layer, plus securing HTTPS traffic if you want it.
 
-These scenarios don't cover every situation, but provide a framework for
-securing {es} and the Elastic Stack based on typical use cases.
-
-NOTE: If you want to enable security by default when you start {es} for the
-first time without manually configuring security, refer to
-<<configuring-stack-security,Starting the Elastic Stack with security enabled>>.
+NOTE: Make sure that you configure security _before_ you start your {es} nodes
+and the auto-configuration process will respect your security configuration.
```

Comment:
Should we add here that TLS configuration can be adjusted or replaced at any time ? I don't want to overcomplicate things but I don't want that users feel that it's either we do it for them or they do it _before_ they start the nodes for the first time or everything is set in store afterwards :) Maybe point to the docs we have for certificate rotation here too ?

Reply:
I think that we can modify the text to indicate that if you configure security before starting ES, we'll respect that configuration. Something like:

>If you configure security manually _before_ starting your {es} nodes, the auto-configuration process will respect your security configuration. You can adjust your TLS configuration at any time, such as <<update-node-certs,updating node certificates>>.